wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Service.kt

```diff
@@ -21,46 +21,28 @@ import com.squareup.wire.schema.ProtoMember.Companion.get
 import com.squareup.wire.schema.ProtoType.Companion.get
 import com.squareup.wire.schema.Rpc.Companion.fromElements
 import com.squareup.wire.schema.internal.parser.ServiceElement
+import kotlin.jvm.JvmName
 import kotlin.jvm.JvmStatic
 
 data class Service(
-  private val protoType: ProtoType,
-  private val location: Location,
-  private val documentation: String,
-  private val name: String,
-  private val rpcs: List<Rpc>,
-  private val options: Options
+  @get:JvmName("type") // For binary compatibility.
```

Comment:
honest question: do we care?

Reply:
(do we care about binary compatibility for wire-schema?)

Reply:
I don't know. I maintained it in the past for a similar migration. It's low effort here so might as well leave it, but I don't really know moving forward. Doesn't Jackson depend on it?